go/cmd/vtgate/status.go

```diff
@@ -160,8 +160,17 @@ isStacked: true,
 
 function update() {
   var varzData;
+
+  // If we're accessing status through a proxy that requires a URL prefix,
+  // add the prefix to the vars URL.
+  var vars_url = '/debug/vars';
+  var pos = window.location.pathname.lastIndexOf('/debug/status');
+  if (pos > 0) {
```

Comment:
If pos=0, then we would be appending to substring(0, 0), which is the empty string. It would give the same result, but it's unnecessary.